doc/private_net.rst

```diff
@@ -0,0 +1,101 @@
+      "networkID" : "0x42",
+      "chainID": "0x42",
+      "allowFutureBlocks" : false
+    },
+    "genesis": {
+      "nonce": "0x0000000000000042",
+      "difficulty": "0x100000",
+      "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+      "author": "0x0000000000000000000000000000000000000000",
+      "timestamp": "0x5A5B92D7",
+      "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+      "extraData": "0x655741534d2074657374206e6574776f726b2030",
+      "gasLimit": "0x989680"
+    },
+    "accounts": {
+      "0000000000000000000000000000000000000001": { "precompiled": { "name": "ecrecover", "linear": { "base": 3000, "word": 0 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000002": { "precompiled": { "name": "sha256", "linear": { "base": 60, "word": 12 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000003": { "precompiled": { "name": "ripemd160", "linear": { "base": 600, "word": 120 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000004": { "precompiled": { "name": "identity", "linear": { "base": 15, "word": 3 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000005": { "precompiled": { "name": "modexp" }, "balance": "0x01" },
+      "0000000000000000000000000000000000000006": { "precompiled": { "name": "alt_bn128_G1_add", "linear": { "base": 500, "word": 0 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000007": { "precompiled": { "name": "alt_bn128_G1_mul", "linear": { "base": 40000, "word": 0 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000008": { "precompiled": { "name": "alt_bn128_pairing_product" }, "balance": "0x01" },
+      "008a78302c6fe24cc74008c7bdae27b7243a7066" /* <= Enter your account address here */: {
+          "balance" : "0x200000000000000000000000000000000000000000000000000000000000000"
+      }
+    }
+  }
+  
+  
+This includes all forks activated at genesis, all precompiled contracts existing on the main net and initial balance for your account.
+
+In case this format gets out of date, please consult the format of the source files in https://github.com/ethereum/aleth/tree/master/libethashseal/genesis.
+
+**3.** Run aleth with ``config.json``.
+::
+  aleth/aleth --config /Users/johndoe/Documents/config.json -m on -a 008a78302c6fe24cc74008c7bdae27b7243a7066 --no-discovery --pin --unsafe-transactions
+
+``-m on`` enables CPU mining;
+
+``-a 008a78302c6fe24cc74008c7bdae27b7243a7066`` sets the beneficiary of the mined blocks;
+
+``--no-discovery --pin`` effectively disables networking; we have ony single node, we don't need to discover other ones and we don't allow others to connect to us;
+
+``--unsafe-transactions`` disables additional prompt before sending each transaction, we don't need it in the testing environment.
+
+It can take a while to mine the first block, but the mining should go faster further on.
```

Comment:
"but the mining should go faster further on" -> "but block latency should decrease with time" (might be nice to have an estimate as to how long this takes e.g. 5 blocks, 100 blocks or 5 minutes, 10 minutes, etc)